src/test/java/com/lpvs/service/GitHubServiceTest.java

```diff
@@ -0,0 +1,376 @@
+package com.lpvs.service;
```

Comment:
Copyright statement should be included at the top of every single java file. Please refer to the case here. [src/main/java/com/lpvs/service/GitHubService.java](https://github.com/Samsung/LPVS/pull/19/files#diff-c788a0a699a9d423378bc544766e7a9bb989247c0db6f8ef4a999ea9e3828609)

Reply:
Thank you for remark. 
Copyright statement was added.

Reply:
Resolved my comment. I will approve this PR after the discussion below. 😄